CMakeLists.txt

```diff
@@ -15,6 +14,10 @@ endif()
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+set(BUILD_TESTS OFF)
+set(BUILD_EXAMPLES OFF)
+set(BUILD_TOOLS OFF)
```

Comment:
These should probably be `CACHE` variables, right? Then there's a way for the user to set it in the configuration, including via `ccmake`.

Reply:
Yes, that should be a CACHE variable, didn't notice earlier... maybe with `option(BUILD_TESTS "Build bundled tests" OFF)` (https://stackoverflow.com/q/36358217/12339402)

Reply:
And, btw, these changes were made by @quiga, so the credit goes to him. I just found it made it easier to include so created this PR upstream.

Reply:
Sorry yeah, I meant `option()` (for ON/OFF variables).

Reply:
Also, just to say, I can't edit the code, I created the PR from quiga's branch, maybe you being a maintainer can make these changes :D